CONTRIBUTORS.md

```diff
@@ -1385,3 +1385,4 @@ Symfony is the result of the work of many people who made the code better
  - Erik Saunier (snickers)
  - Matej Žilák (teo_sk)
  - Vladislav Vlastovskiy (vlastv)
+ - Almog Baku (AlmogBaku)
```

Comment:
No need to add yourself to this file because its contents are automatically generated. And you'll appear here too: http://symfony.com/contributors/code